onlineconf.go

```diff
@@ -8,28 +8,67 @@ import (
 	"fmt"
 	"io"
 	"log"
+	"os"
+	"path"
 	"strconv"
 	"sync"
+	"syscall"
 
+	"github.com/alldroll/cdb"
 	"github.com/fsnotify/fsnotify"
-	"github.com/jbarham/go-cdb"
+	"github.com/grandecola/mmap"
 )
 
-const configDir = "/usr/local/etc/onlineconf"
+var DefaultConfigDir = "/usr/local/etc/onlineconf"
```

Comment:
There is no way to redefine a default. So the naming is incorrect. This should be named just `ConfigDir`.

Nonetheless changing this value after `initWatcher()` was called will break the watcher.
Instead I recommend to add public `Initialize(path)`, panic when it is called the second time and call `Initialize(configPath)` automatically from `newModule()` if not yet.

Reply:
Renamed. `Initialize(path)` added.

> Initialize(configPath) automatically from newModule() if not yet.

I don't see the need